tests/TestRunner/app/Inheritance/InheritanceTests.js

```diff
@@ -1846,39 +1846,54 @@ describe(module.id, function () {
         expect(NSString.stringWithCString(encoding).toString()).toBe("v@:");
     });
 
-    it("should project Symbol.iterable as NSFastEnumeration", function () {
-        var start = 1;
-        var end = 10;
-        var lastStep = 0;
-        var IterableClass = NSObject.extend({
-            [Symbol.iterator]() {
-                return {
-                    step: start,
-
-                    next() {
-                        if (this.step <= end) {
-                            return { value: this.step++, done: false };
-                        } else {
-                            return { done: true };
-                        }
-                    },
-
-                    return() {
-                        lastStep = this.step;
-                        return {};
-                    }
-                }
+    it("should project Symbol.iterable as NSFastEnumeration", function() {
```

Comment:
I think we don't use clang-format for JS files: https://github.com/NativeScript/ios-runtime/blob/e34fe1094df0868eb34147c06276b421fffb0493/tools/pre-commit-clang-format#L33

But if this is the right indentation, I'm all in for that.